src/ConnectedMode.UnitTests/Migration/FileCleanerTestCases/AdditionalFiles_my_project_key_Cleaned.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The tests use embedded files for the input files and the expected output.

Having them inline in the tests makes the tests hard to read. Also, having them on the disc makes them easier to diff.